testing/batch_processing/cached_results/t2/csa_pam50_PAM50.csv

```diff
@@ -0,0 +1,885 @@
+Timestamp,SCT Version,Filename,Slice (I->S),VertLevel,DistancePMJ,MEAN(area),STD(area),MEAN(angle_AP),STD(angle_AP),MEAN(angle_RL),STD(angle_RL),MEAN(diameter_AP),STD(diameter_AP),MEAN(diameter_RL),STD(diameter_RL),MEAN(eccentricity),STD(eccentricity),MEAN(orientation),STD(orientation),MEAN(solidity),STD(solidity),SUM(length)
+"2023-01-19 14:37:23","git-sb/3733-bring-metrics-to-pam50-786f6302859e392e257a498576df5a0f55c2315e","/home/GRAMES.POLYMTL.CA/sebeda/spinalcordtoolbox/data/sct_example_data/t2/t2_seg.nii.gz","80","20","","","","","","","","","","","","","","","","","",""
+"2023-01-19 14:37:23","git-sb/3733-bring-metrics-to-pam50-786f6302859e392e257a498576df5a0f55c2315e","/home/GRAMES.POLYMTL.CA/sebeda/spinalcordtoolbox/data/sct_example_data/t2/t2_seg.nii.gz","81","20","","","","","","","","","","","","","","","","","",""
+"2023-01-19 14:37:23","git-sb/3733-bring-metrics-to-pam50-786f6302859e392e257a498576df5a0f55c2315e","/home/GRAMES.POLYMTL.CA/sebeda/spinalcordtoolbox/data/sct_example_data/t2/t2_seg.nii.gz","82","20","","","","","","","","","","","","","","","","","",""
+"2023-01-19 14:37:23","git-sb/3733-bring-metrics-to-pam50-786f6302859e392e257a498576df5a0f55c2315e","/home/GRAMES.POLYMTL.CA/sebeda/spinalcordtoolbox/data/sct_example_data/t2/t2_seg.nii.gz","83","20","","","","","","","","","","","","","","","","","",""
+"2023-01-19 14:37:23","git-sb/3733-bring-metrics-to-pam50-786f6302859e392e257a498576df5a0f55c2315e","/home/GRAMES.POLYMTL.CA/sebeda/spinalcordtoolbox/data/sct_example_data/t2/t2_seg.nii.gz","84","20","","","","","","","","","","","","","","","","","",""
```

Comment:
I notice that the generated `_PAM50.csv` file has a lot of extra rows:

- the PAM50 template spans `VertLevel = 1-20`
- `t2.nii.gz` spans only `VertLevel 2-5`
- So, most of the columns are blank, because `VertLevel 1` and `VertLevel 6-20` have no metrics.

Is there a way to generate this CSV file without generating the unnecessary rows?

Reply:
the way to do it is to specify `-vert` ironically haha! But in the context we want to use it, it is nice to have the exact same size of .csv file --> we want to use the normalized values in a large database to then compare with patients

Reply:
> But in the context we want to use it, it is nice to have the exact same size of .csv file --> we want to use the normalized values in a large database to then compare with patients

Ahhhh, I see! I suppose it is a little extra work to manipulate the CSV files if they're all different sizes... :thinking:

Reply:
hmm, we could force the parameter `-vert 1:20` when we specify `normalize-PAM50 1` since it would still have the same size of .csv files

Reply:
Just out of curiosity, what kind of data analysis tool would you be using for subject comparison of the csv files in your project?

Say the csv files are different sizes -- couldn't you like, do the same thing that this PR is doing now (initializing all 991 slices with `np.nan`, then copying over values slice-by-slice), but move that init-and-copy logic into your analysis script?

Reply:
the idea here, we want to create a database of healthy suvjects with their metrics in the normative PAM50 space, so we will use it, but the idea would be to make it public, that's why we discussed with julien that it would be better to have the same format for all files! (we use it in #4003 by the way!)

Reply:
Ahhhh, gotcha!

Just to clarify, what do you mean by "database"? Just like, a collection of output CSV files in a repo? Or, something more structured?

Reply:
For now just a collection of CSV files (not even on a repo for now), we will also include a participants.tsv with information on the subjects, or now we are populating it! Maybe when we are more advance we will have something more sophisticated!

Reply:
Ahhh, it makes perfect sense to want to have extra rows, then.

Hmm. What if we used `-verbose 2` for this? For example:

```diff
    if normalize_pam50:
        fname_vert_level_PAM50 = os.path.join(__data_dir__, 'PAM50', 'template', 'PAM50_levels.nii.gz')
-       metrics_PAM50_space = interpolate_metrics(metrics, fname_vert_level_PAM50, fname_vert_level)
+       keep_blank_rows = True if verbose == 2 else False
+       metrics_PAM50_space = interpolate_metrics(metrics, fname_vert_level_PAM50, fname_vert_level, keep_blank_rows)
        metrics = metrics_PAM50_space  # Set metrics to the metrics in PAM50 space to use instead
        fname_vert_level = fname_vert_level_PAM50  # Set vertebral levels to PAM50
```

That way, you can specify `-v 2` for your project to get the "full" output, but the default behavior (`-v 1`) is to output only the slices that are available in the image (which I think matches user expectations for `sct_process_segmentation`).

Reply:
yes we could do that, but this CSV file would also be an input of the updated version of `sct_compute_mscc` in #4003 just to consider! (if we do this just to make it compatible with the other feature)

Reply:
Thank you for the heads up! I haven't looked at #4003 yet, but I'll take some time to read it over and make sure I understand it before proposing further changes.